test/classes/bradc/compilerErrorInMethod/testClear-snap1.chpl

```diff
@@ -57,5 +57,11 @@ var a = new Wrap(_value = new Assoc());
 if doDense {
   d = 1;
 }
+
 s = 1;
 a = 1;
+
+delete a._value;
+delete s._value;
+delete d._value;
```

Comment:
While this will resolve the leaks, should they go in some sort of destructor method? It's probably OK to go in as is, since the test doesn't really intend to use memory management.